decidim-sortitions/app/views/decidim/sortitions/sortitions/_tags.html.erb

```diff
@@ -1,4 +1,4 @@
-<ul class="tags tags--sortition">
+<ul class="tags--sortition">
```

Comment:
I think `tags-sortition` should be in the erblint configuration too. I don't find where it's defined/used

Reply:
done in [c4ef592](https://github.com/decidim/decidim/pull/12081/commits/c4ef592fccb97f036bda93b315302ec7661b3070)